packages/sources/dlc-btc-por/src/transport/proof-of-reserves.ts

```diff
@@ -210,7 +210,15 @@ export class DLCBTCPorTransport extends SubscriptionTransport<TransportTypes> {
       txId,
       requestConfig,
     )
-    return response.data.result
+
+    const result = response.data.result
+
+    if (!result) {
+      logger.error('BITCOIN_RPC_URL - getrawtransaction failed, requestConfig: ', requestConfig)
```

Comment:
We don't need the `requestConfig` but maybe only `txId`

Reply:
In a current exchange with a NOP on this, printing out the `requestConfig` might have highlighted the situation faster. However, it isn't a huge value add either.

Reply:
It looks like they have issues with their RPC. Do you still think this change will add a value ?

Reply:
I do think it adds, value as their RPC actually follows the jsonrpc spec perfectly, and it wasn't an uptime issue. So it is technically an issue with the EA not with them. The changes they are making now to accommodate the missing id field in their RPC is a work around. Even though most RPCs can handle a missing id field, we should probably support all RPCs which technically follow the spec.